src/hooks/constants/lib/fetchConstants.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants } from '../../../generated/graphql';
+import { fetchConstants } from './fetchConstants';
+import {
+  getMockApiPromise,
+  mockedRepayFee,
+} from '../../polkadotJs/tests/mockUsePolkadotJsContext';
+
+describe('fetch chain constants', () => {
+  let mockApiInstance: ApiPromise;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
create react app should be doing this automatically already, not clear but actually reset:
https://github.com/facebook/create-react-app/blob/main/packages/react-scripts/scripts/utils/createJestConfig.js#L68

Reply:
No, lets keep this one clean